app/webpacker/controllers/flatpickr_controller.js

```diff
@@ -0,0 +1,110 @@
+// import Flatpickr
+import Flatpickr from "stimulus-flatpickr";
+import { ar } from "flatpickr/dist/l10n/ar";
+import { cat } from "flatpickr/dist/l10n/cat";
+import { cy } from "flatpickr/dist/l10n/cy";
+import { de } from "flatpickr/dist/l10n/de";
+import { fr } from "flatpickr/dist/l10n/fr";
+import { it } from "flatpickr/dist/l10n/it";
+import { nl } from "flatpickr/dist/l10n/nl";
+import { pl } from "flatpickr/dist/l10n/pl";
+import { pt } from "flatpickr/dist/l10n/pt";
+import { ru } from "flatpickr/dist/l10n/ru";
+import { sv } from "flatpickr/dist/l10n/sv";
+import { tr } from "flatpickr/dist/l10n/tr";
+import { en } from "flatpickr/dist/l10n/default.js";
```

Comment:
I bet that we forget this when adding a new language. Wouldn't it be better to use `I18n.available_locales` here?

Reply:
Yes, I'm pretty sure too. But adding a new language on OFN doesn't mean it exists on flatpickr. So we might have some trouble using automatic logic...

Reply:
In that case we need a list of available flatpickr translations as well and then we select from that.

Reply:
Maybe worth a new issue/PR? Because it's not that easy here (dynamic import, list of available languages, ...)